src/commands/plugins.rs

```diff
@@ -104,7 +108,14 @@ impl Install {
         // Downgrades are only allowed via the `upgrade` subcommand
         let downgrade = false;
         let manifest = manager.get_manifest(&manifest_location).await?;
-        try_install(&manifest, &manager, self.yes_to_all, downgrade).await?;
+        try_install(
+            &manifest,
+            &manager,
+            self.yes_to_all,
```

Comment:
Wondering if the time has come to put the yes, OCC and downgrade flags into an InstallOptions type... (definitely not a must-do at this point though!)

Reply:
Good point -- this would ensure consistency across subcommands. Let's save this for another PR.